ui/src/helpers/rules.js

```diff
@@ -1,5 +1,5 @@
-const required = (value = '', message = 'Campo obrigatório') => {
-  return String(value).length || message
+function required (value = '', message = 'Campo obrigatório') {
+  return !!value || message
 }
```

Comment:
fez todos os testes? pq dessa for, o `0` sera false

Reply:
Feito